### ISSUE TITLE
Change name to `sortingview-plugin` or include in `sortingview` repo?

### ISSUE THREAD

**Issue by alejoe91** [magland/spikesortingview#158]
@magland 

Since this is basically the visualization plugin for `sortingview`, I think that the naming is very confusing (as you mention in the README).

What about `sortingview-plugin`? 

Alternatively, do you see any drawback in including the package into a `plugin` folder in the `sortingview` repo itself? This could be a template also for other plugins, to make sure that the Python and the TS/JS are in sync. One could also imagine to add GH actions that install and deploy both packages locally

**Comment by magland** [comment]
@alejoe91 I agree this is not an ideal name. However, this particular package is not under development at this point. In the medium term this will be replaced by a different package (although this package and existing figurl's will continue to work).
